scripts/enderio/eio_armor_weapons.zs

```diff
@@ -154,12 +154,10 @@ recipes.addShaped("eio_step_explosive_upgrade_5", <enderio:item_dark_steel_upgra
 
 // Dark Steel Helmet
 assembler.recipeBuilder()
-	.inputs(circuits[2] * 2)
 	.inputs(<ore:plateDarkSteel> * 5)
-	.inputs(<ore:foilPlastic> * 5)
-	.inputs(<ore:wireFineCopper> * 10)
+	.inputs(circuits[2] * 2)
     .inputs(<ore:screwDarkSteel> * 4)
-    .fluidInputs([<fluid:soldering_alloy> * 576])
+	.notConsumable(scripts.gregtech_globals.intCircuit(5))
```

Comment:
For these recipes, you can simply do `.circuit(5)`. It's a method provided by GTCE directly.